apis/apis.json

```diff
@@ -5729,7 +5730,8 @@
       "shortName": "containeranalysis",
       "serviceConfigFile": "grafeas_v1.yaml",
       "restNumericEnums": true,
-      "includeCommonResourcesProto": true
+      "includeCommonResourcesProto": true,
+      "transport": "grpc+rest"
```

Comment:
Undo this one. This is being reverted, se b/330194871.

Reply:
Done.